tests/tls-issuerdn/test.yaml

```diff
@@ -0,0 +1,17 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
+
+checks:
+  - filter:
+      count: 1
+      match:
+        event_type: tls
+        tls.session_resumed: true
+        
```

Comment:
Suggestion: let's `tls.issuerdn` here, as well, as this also helps to showcase how the output is the same, regardless of using the legacy keyword or the more recent one. ;)